Gruntfile.js

```diff
@@ -85,72 +23,58 @@ module.exports = function(grunt) {
     return file;
   });
 
-  var pluginCombinations = combine(plugins);
-  var pluginConcatFiles = pluginCombinations.reduce(function(dict, comb) {
```

Comment:
There was probably also a reason for building all plugins in all possible combinations.

But I couldn't think of a reason why to do this, so I removed it to make the build simpler.

Reply:
We need this functionality, as we generate all variants of a the plugin+core bundles for our hosted CDN – http://ravenjs.com/